src/acceptance-test/java/uk/gov/ida/verifyserviceprovider/NonMatchingAcceptanceTest.java

```diff
@@ -260,90 +252,6 @@ public void shouldThrowExceptionIfLoAReturnedByIdpIsTooLow () {
-        GenerateRequestService generateRequestService = new GenerateRequestService(client);
-
-        complianceTool.initialiseWithDefaultsForV2();
-
-        RequestResponseBody requestResponseBody = generateRequestService.generateAuthnRequest(applicationWithEidasEnabled.getLocalPort());
-        Map<String, String> translateResponseRequestData = ImmutableMap.of(
-                "samlResponse", complianceTool.createResponseFor(requestResponseBody.getSamlRequest(), VERIFIED_USER_ON_SERVICE_WITH_NON_MATCH_SETTING_ID),
-                "requestId", requestResponseBody.getRequestId(),
-                "levelOfAssurance", LEVEL_1.name()
-        );
-
-        Response response = client
-                .target(String.format("http://localhost:%d/translate-response", applicationWithEidasEnabled.getLocalPort()))
-                .request()
-                .buildPost(json(translateResponseRequestData))
-                .invoke();
-
-        assertThat(response.getStatus()).isEqualTo(OK.getStatusCode());
-
-        JSONObject jsonResponse = new JSONObject(response.readEntity(String.class));
-        assertThat(jsonResponse.getString("scenario")).isEqualTo(IDENTITY_VERIFIED.name());
-        assertThat(jsonResponse.getString("levelOfAssurance")).isEqualTo(LEVEL_1.name());
-    }
-
-    @Test
-    public void shouldProcessIdpResponseCorrectlyWhenEuropeanIdentityDisabled() {
-        Client client = applicationWithEidasDisabled.client();
-        ComplianceToolService complianceTool = new ComplianceToolService(client);
-        GenerateRequestService generateRequestService = new GenerateRequestService(client);
-
-        complianceTool.initialiseWithDefaultsForV2();
-
-        RequestResponseBody requestResponseBody = generateRequestService.generateAuthnRequest(applicationWithEidasDisabled.getLocalPort());
-        Map<String, String> translateResponseRequestData = ImmutableMap.of(
-                "samlResponse", complianceTool.createResponseFor(requestResponseBody.getSamlRequest(), VERIFIED_USER_ON_SERVICE_WITH_NON_MATCH_SETTING_ID),
-                "requestId", requestResponseBody.getRequestId(),
-                "levelOfAssurance", LEVEL_1.name()
-        );
-
-        Response response = client
-                .target(String.format("http://localhost:%d/translate-response", applicationWithEidasDisabled.getLocalPort()))
-                .request()
-                .buildPost(json(translateResponseRequestData))
-                .invoke();
-
-        assertThat(response.getStatus()).isEqualTo(OK.getStatusCode());
-
-        JSONObject jsonResponse = new JSONObject(response.readEntity(String.class));
-        assertThat(jsonResponse.getString("scenario")).isEqualTo(IDENTITY_VERIFIED.name());
-        assertThat(jsonResponse.getString("levelOfAssurance")).isEqualTo(LEVEL_1.name());
-    }
-
-    @Test
-    public void shouldProcessIdpResponseCorrectlyWhenEuropeanIdentityConfigAbsent() {
```

Comment:
This test seems to be exercising the non eIDAS app - should it be kept?

Reply:
I've restored, thanks!